solutions/object_oriented_design/parking_lot/parking_lot.py

```diff
@@ -111,10 +109,10 @@ def __init__(self, level, row, spot_number, spot_size, vehicle_size):
         self.vehicle = None
 
     def is_available(self):
-        return True if self.vehicle is None else False
+        return not self.vehicle
 
     def can_fit_vehicle(self, vehicle):
-        if self.vehicle is not None:
+        if self.vehicle:
```

Comment:
I do agree this is an optimized option, but the other version is a bit more explicit which may help with the readability of it.